notebooks/scrubbed/covid-19-tests-august-2020.ipynb

```diff
@@ -0,0 +1,599 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metaculus = ergo.Metaculus(username=\"oughtpublic\", password=\"123456\", api_domain=\"pandemic\")\n",
+    "ventilators_question = metaculus.get_question(4400, name=\"# tests needed\")\n",
+    "ventilators_question.show_community_prediction()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is similar to our prediction, just less certain (longer tails) and centered slightly higher. Since I think they have valuable information I might have missed, I'm going to create a simple essemble prediction: pick randomly between a prediction the Metaculus community would make, and I would make, to create a combined model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ensemble_prediction():\n",
+    "    \"# ensemble prediction\"\n",
+    "    if ergo.flip(0.5):\n",
+    "        return ventilators_question.sample_community()\n",
+    "    else:\n",
+    "        return test_results_for_deadline_with_slope()\n",
+    "\n",
+    "show_marginal(ensemble_prediction)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This shows a distribution similar to the one we had, just less certain, and with some probability for the long tails. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Submitting to metaculus\n",
+    "\n",
+    "If we want to, we can now submit this model to Metaculus."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# below commented out to avoid accidentally submitting an answer when you don't mean to\n",
+    "# samples = ergo.run(ensemble_prediction, num_samples=1000)[\"output\"]\n",
+    "# ventilators_question.submit_from_samples(samples)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Summary\n",
+    "So create this model, we did the following:\n",
+    "- we fetched the latest data from https://covidtracking.com/\n",
+    "- estimated the growth of the number of tests to be a combination of the past linear growth, and a log-normal distribution\n",
+    "- used that to extrapolate the number of tests we'd have in August\n",
+    "- then combined that prediction with the metaculus prediction, to create a final forecast for number of tests on August 1st\n",
+    "\n",
+    "Since we did a lot of iteration on our model, I'm including a cleaned up version of the entire thing below, so you can see everything at once."
```

Comment:
That's very fair. Since this is functioning mostly as a tutorial (I don't expect to make major updates, and if I do, they'll be in a new notebook), I'm ok with the duplication since I think it adds clarity.